import os, re

def fail():
    print 'this code has some fail in it, like this long long line'
